package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ecosia-privacy-policy",
+  "version": "1.0.0",
+  "description": "Ecosia's Privacy Policy in multiple languages using markdown",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Since this is no code, I would actually remove this. Otherwise it looks like a big glaring TODO.